### PR TITLE
Feature/divider component

### PR DESCRIPTION
**Pull request by kyleturco**
Add a `<Divider />` component and examples for the divider. (Note: currently failing the Wercker test as a result of the Babel update)

### CODE REVIEW

File: docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <Divider className='vertical'>And</Divider>
+        <div className='column'>
+          <Segment className='basic'>
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec odio.
+          </Segment>
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (kyleturco) [inline docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js:28]: Does SD have a `<Column>` component? I used regular `<div>` and set the className to column in this example. We do need the columns though to get the proper formatting to show the vertical dividers

File: docs/app/Examples/elements/Divider/DividerExamples.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import DividerTypesExamples from './Types/DividerTypesExamples';
+import DividerVariationsExamples from './Variations/DividerVariationsExamples';
+
+export default class ButtonExamples extends Component {
```
Comment (levithomason) [inline docs/app/Examples/elements/Divider/DividerExamples.js]: Class name >.<

Comment (kyleturco) [inline docs/app/Examples/elements/Divider/DividerExamples.js]: Oh no, it never stops

File: docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import {Segment, Divider} from 'stardust';
+
+export default class DividerVerticalExample extends Component {
+  render() {
+    return (
+      <div className='ui three column relaxed grid'>
+        <div className='column'>
```
Comment (levithomason) [inline docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js]: We can use Stardust Grid and Columns here.

File: docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js (hunk 1 above)
Comment (levithomason) [inline docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js:28]: Oops, comments on this above before seeing your comment.  Yep, just add `Grid, Column` to the import from stardust.  Like other components, they include all the necessary base classes so you only need to add modifier classes (like `three column relaxed` to the `<Grid />`).

**Comment by levithomason** [comment]
After updating the one class name and using `Grid` and `Column` components, :ship: !

File: docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js (hunk 1 above)
Comment (kyleturco) [inline docs/app/Examples/elements/Divider/Types/DividerVerticalExample.js:28]: Roger that, both `Column` and `Grid` work 👍

**Pull request merged**